tests/client_no_stream_test.go

```diff
@@ -39,3 +40,11 @@ func TestStatsNoStream(t *testing.T) {
 		t.Error("No stream block should result in StreamZoneSync = `nil`")
 	}
 }
+
+func getTestEndpoint() string {
+	ep := os.Getenv("NGINX_TEST_ENDPOINT")
+	if ep != "" {
+		return ep
+	}
+	return "http://127.0.0.1:8080/api"
```

Comment:
Since we're defining defaults in the Makefile, it doesn't make sense to define them here as well.  If the env is not set, you can panic here:
```
	panic("NGINX_TEST_ENDPOINT env variable is not set or empty")
```

the comment applies to the other similar functions

Reply:
Hm. I did this to give the option of doing:
```console
$ make run-nginx-plus
$ go test tests/client_test.go
```

I suppose I'm fine with forcing:
```console
$ make run-nginx-plus
$ make test-run
```